tests/Model/TypeMapArrayIteratorTest.php

```diff
@@ -2,6 +2,7 @@
 
 namespace MongoDB\Tests\Model;
 
+use MongoDB\Exception\BadMethodCallException;
```

Comment:
`@expectedException` takes a fully qualified class name so we don't actually need to `use` it here.